ucp/_libs/ucp_py.pyx

```diff
@@ -46,7 +46,7 @@ def handle_msg(msg):
 
     Parameters
     ----------
-    msg : ucp_msg
+    msg : UCPMessage
```

Comment:
Since we're going from `ucp_ep` to `Endpoint`, should we just move from `ucp_msg` to `Message`.   Inferring the kind of message from the namespace should suffice right?

Reply:
Yeah - if we are dumping the `ucp_` convention, `Message` should suffice.